Resources/Private/Templates/FrontEndEditor/Index.html

```diff
@@ -17,6 +17,8 @@ <h2>
                         <th scope="col">
                             <f:translate key="plugin.frontEndEditor.property.title"/>
                         </th>
+                        <th scope="col">
+                        </th>
```

Comment:
Should we add a label as well? Something like "Actions"?

Reply:
Good idea! Done.